### ISSUE TITLE
Imports .custommodule breaks on Python 3

### ISSUE THREAD

**Issue by PeterHayward21** [jhpyle/docassemble#122]
Given

```
---
imports:
  - .customModule
---
```
Error:

```
Traceback (most recent call last):
  File "/usr/share/docassemble/local3.5/lib/python3.5/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/share/docassemble/local3.5/lib/python3.5/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/share/docassemble/local3.5/lib/python3.5/site-packages/docassemble/webapp/server.py", line 6392, in index
    interview.assemble(user_dict, interview_status, old_user_dict)
  File "/usr/share/docassemble/local3.5/lib/python3.5/site-packages/docassemble/base/parse.py", line 4674, in assemble
    exec('import ' + str(self.source.package) + module_name, user_dict)
  File "<string>", line 1, in <module>
ImportError: No module named 'docassemble.playground1'
```

This used to work with DA before the Python 3 transition.

**Comment by jhpyle** [comment]
Sorry about that -- it will be fixed in the next version.  This only affects the Playground, by the way.  If you install the package on the system it will work.

**Comment by jhpyle** [comment]
This is fixed in 0.4.16.

**Comment by PeterHayward21** [comment]
Thank you very much!
